src/components/TableCard.tsx

```diff
@@ -0,0 +1,102 @@
+import { ReactNode, useMemo } from "react";
+import { useHover } from "react-aria";
+import { Css, Palette } from "src/Css";
+import { ButtonMenu, MenuItem } from "./ButtonMenu";
+import { Tag, TagType } from "./Tag";
+
+export type TableCardType = "card" | "list";
+
+type CardTag = {
+  text: string;
+  type?: TagType;
+};
+
+export interface TableCardProps {
+  title: string;
+  subtitle: string;
+  metadata: ReactNode;
+  image: string;
+  // contain displays entire image, cover fills the space
+  imageFit?: "contain" | "cover";
+  type?: TableCardType;
+  bordered?: boolean;
+  disabled?: boolean;
+  buttonMenuItems?: MenuItem[];
+  tag?: CardTag;
+}
+
+export function TableCard(props: TableCardProps) {
+  const {
+    title,
+    subtitle,
+    metadata,
+    image,
+    imageFit = "contain",
```

Comment:
Cool idea thinking about the image fit behavior - I'm wondering though how practical exposing this to the caller is though? Like the expectation is these cards are going to be used (probably as a list) to present user uploaded images of various sizes where the implementor won't be able to pre-optimize for what that image is going to be. Unless you have a specific use case in mind where a page will surely need `cover` image fit type?

Reply:
Yeah, good point. Obviously for items we wouldn't want part of the image cut off, but it seemed like maybe for the plan images they might want them to cover the whole area (from the design example), but perhaps it does make more sense to just leave it as contain to keep it simple. I can ask Oz about his expectation.

This is the plan image with contain vs cover. I was just trying to get something working to match the design.

<img width="286" alt="image" src="https://github.com/user-attachments/assets/b423fe3c-8248-48b6-b7f3-6d3ef35d94c2" />
<img width="277" alt="image" src="https://github.com/user-attachments/assets/fcd70018-40f3-433f-acb3-66275a9b8c7c" />